user/libpinyin/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "libpinyin"
+pkgver = "2.8.1"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr", "--with-dbm=KyotoCabinet", "--enable-libzhuyin"]
+hostmakedepends = ["pkgconf"]
+makedepends = ["kyotocabinet", "glib-devel"]
+depends = ["pcre2-devel"]
+pkgdesc = "Algorithms core for intelligent sentence-based Chinese pinyin IMEs"
+maintainer = "metalparade <comer@live.cn>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/libpinyin/libpinyin"
+source = f"{url}/releases/download/{pkgver}/libpinyin-{pkgver}.tar.gz"
+sha256 = "353154f06d71dd0737b77ddcb27cb0dcaddb00f7ccd695bc0314bb42050e9050"
+options = ["!lto", "!autosplit"]
```

Comment:
these options are wrong for both of these things and there's no comment explaining why they are set

Reply:
like, libraries normally have a -devel package that is split, and autosplit is always required for the default splits, so why are both missing and the autosplit disabled?

Reply:
if you're not sure what this means, look at other libraries packaged like `main/libass` which shows everything correctly